contrib/functions/go/generate-blueprint-docs/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt-functions-catalog/contrib/functions/go/blueprint-docs/docs"
+	"github.com/GoogleContainerTools/kpt-functions-catalog/contrib/functions/go/blueprint-docs/generated"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+const defaultReadmePath = "/tmp/README.md"
+const defaultRepoPath = "https://github.com/GoogleCloudPlatform/blueprints.git/catalog/"
+
+//nolint
+func main() {
+	rp := ReadmeProcessor{}
+	cmd := command.Build(&rp, command.StandaloneEnabled, false)
+
+	cmd.Short = generated.GenerateBlueprintDocsShort
+	cmd.Long = generated.GenerateBlueprintDocsLong
+	cmd.Example = generated.GenerateBlueprintDocsExamples
+	if err := cmd.Execute(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
No need to print again here. `command.Build(&rp, command.StandaloneEnabled, false)` is doing it for us. The last argument is to enable it.

Reply:
done